### ISSUE TITLE
if-else chain UI

### ISSUE THREAD

**Issue by FallenDemon2000** [andre-santos-pt/javardise#37]
Reparei que o Javardise não suporta o if-else usual e acho que seria um upgrade se pudesse suportar, quando se põe IFs dentro de else ja não fica tão bonito como podia ser.

**Example:**
![image](https://github.com/andre-santos-pt/javardise/assets/61288796/84d8df2f-ca19-4fa0-bd86-329fc644a967)

**Comment by andre-santos-pt** [comment]
totalmente de acordo!.. apenas a forma de edição não é óbvia, mas é certamente uma melhoria para o futuro.

**Comment by FallenDemon2000** [comment]
> totalmente de acordo!.. apenas a forma de edição não é óbvia, mas é certamente uma melhoria para o futuro.

Exato, eu tive algumas dificuldades em vez como meter ai o else. Não me pareceu muito obvio
